include/boost/geometry/formulas/vertex_longitude.hpp

```diff
@@ -164,27 +175,104 @@ class vertex_longitude_on_spheroid
 
 };
 
+//CS_tag dispatching
 
 template <typename CT, typename CS_Tag>
-struct vertex_longitude
+struct compute_vertex_lon
 {
     BOOST_MPL_ASSERT_MSG
-         (
-             false, NOT_IMPLEMENTED_FOR_THIS_COORDINATE_SYSTEM, (types<CS_Tag>)
-         );
+    (
+            false, NOT_IMPLEMENTED_FOR_THIS_COORDINATE_SYSTEM, (types<CS_Tag>)
+            );
```

Comment:
There are some changes in indentation which seems to be mistakes or something automatic, also above.